src/presentational/AssignmentForm.jsx

```diff
@@ -0,0 +1,144 @@
+import { Component, PropTypes } from 'react';
+import { Link } from 'react-router';
+
+import InputElement from './InputElement';
+
+const initialState = {
+  classifications_target: '',
+  description: '',
+  duedate: '',
+  name: '',
+  students: [],
+  subjects: [],
+}
+
+
+class AssignmentForm extends Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.renderStudentList = this.renderStudentList.bind(this);
+    this.toggleStudent = this.toggleStudent.bind(this);
+    this.state = Object.assign({}, initialState);
+  }
+
+
+  toggleStudent(id) {
+    if (this.state.students.includes(id)) {
```

Comment:
`const { students } = this.state`

then you can remove every instance of `this.state` in this function